homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -375,6 +388,9 @@ def get_entity_state(config, entity):
                 final_brightness = 170
             elif speed == SPEED_HIGH:
                 final_brightness = 255
+        elif entity.domain == "cover":
```

Comment:
cover.DOMAIN

Reply:
doesn't work

Reply:
of course it works... just need to import